spec/functional/application_spec.rb

```diff
@@ -26,12 +26,12 @@
   let(:argv) { [] }
   let(:stderr) { StringIO.new }
 
-  before(:each) do
+  before do
```

Comment:
i'm not sure i'm wild about this one, makes it easier for newbies to clearly see that `:each` is the default.